docs/manual/idlcxx.rst

```diff
@@ -158,15 +155,15 @@ std::array:
 
 The command line options for custom array containers are:
 
-- array-template
+- ``array-template``
 
   - template to use for arrays instead of std::array
 
   - replaced tags: {TYPE}, {DIMENSION}
```

Comment:
Maybe also make {TYPE} and {DIMENSION} here into ``{TYPE}`` and ``{DIMENSION}`` since we do it in the previous sections too.

Reply:
Makes sense.